metricbeat/module/mysql/vendor/vendor.json

```diff
@@ -3,10 +3,22 @@
 	"ignore": "test github.com/elastic/beats",
 	"package": [
 		{
-			"checksumSHA1": "42vkdsxNaLyPu+FktCzZ/8zsNSE=",
+			"path": "appengine/cloudsql",
```

Comment:
I wonder why this entry does not have a revision but is still here?

Reply:
Umm, we have [another case](https://github.com/elastic/beats/blob/b1afbf107ae122bdda41541d3e3f8fea6eb093e8/vendor/vendor.json#L2459) in the main one. This was added by govendor, maybe because it appears in an import in the `google.golang.org/appengine/cloudsql` package itself. :thinking: